MonoGame.Framework/Matrix.cs

```diff
@@ -715,7 +1008,16 @@ public static void CreatePerspectiveFieldOfView(float fieldOfView, float aspectR
 		    result.M43 = (nearPlaneDistance * farPlaneDistance) / (nearPlaneDistance - farPlaneDistance);
         }
 
-
+        /// <summary>
+        /// Creates a new projection <see cref="Matrix"/> for customized perspective view.
+        /// </summary>
+        /// <param name="left">Left coordinate of the viewing field.</param>
+        /// <param name="right">Right coordinate of the viewing field.</param>
+        /// <param name="bottom">Bottom coordinate of the viewing field.</param>
+        /// <param name="top">Top coordinate of the viewing field.</param>
+        /// <param name="nearPlaneDistance">Near plane of the viewing field.</param>
+        /// <param name="farPlaneDistance">Far plane of the viewing field.</param>
+        /// <returns>The new <see cref="Matrix"/> for customized perspective view.</returns>

```

Comment:
The `of the viewing field` repetition here bothers me....  it feels like filler and we need more useful information.

@KonajuGames ?